exasol_integration_test_docker_environment/lib/base/db_os_executor.py

```diff
@@ -39,6 +42,9 @@ class DbOsExecutor(Protocol):
     def exec(self, cmd: str) -> ExecResult:
         ...
 
+    def prepare(self):
+        pass
+
 
 class DockerExecutor(DbOsExecutor):
```

Comment:
the docker executor simply uses the prepare method of the protocol, I guess

Reply:
Yes. We could also use an abstract method and implement it in both places (`SshExecutor` and `DockerExecutor`).
I currently preferred the shorter version but I am open to your proposals.